src/build.js

```diff
@@ -9,7 +9,12 @@ var defaults = require('./defaults.js'),
     solveOutput = require('./solveOutput.js'),
     createCompiler = require('./createCompiler.js'),
     createLogger = require('./createLogger.js'),
-    _ = require('lodash');
+    _ = require('lodash'),
+    exitStatus = 0;
```

Comment:
Do you really need to externalize the exit status here? Can't you just fire process.exit in the relevant conditions? Plus it seems kinda weird to have this status external to the function and thus global in a sense.